lib/pre_build.ts

```diff
@@ -20,6 +21,13 @@ export async function runPreBuild(
 ): Promise<PreBuildOutput> {
   const home = Deno.env.get("HOME");
   const root = Deno.cwd();
+  if (!await pathExists(path.join(root, "Cargo.toml"))) {
+    console.error(
+      "%cConsider running `wasmbuild new` to get started",
```

Comment:
```suggestion
      "%cConsider running `deno task wasmbuild new` to get started",
```

Maybe let's assume they have it as a task?